packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -385,7 +385,9 @@ class _AuthorizedFlutterPreSubmitComparator extends FlutterPreSubmitFileComparat
     await update(golden, imageBytes);
     final File goldenFile = getGoldenFile(golden);
 
-    return skiaClient.tryjobAdd(golden.path, goldenFile);
+    await skiaClient.tryjobAdd(golden.path, goldenFile);
+
+    return true;
```

Comment:
This probably deserves a comment. Why does the comparison always succeed?

Reply:
Done!